apps/web-giddh/src/app/angular21-compatibility.ts

```
@@ -0,0 +1,207 @@
+        }
+    };
+
+    // Patch setTimeout to catch async factory/onDestroy errors
+    const originalSetTimeout = window.setTimeout;
+    (window as any).setTimeout = function(callback: TimerHandler, delay?: number, ...args: any[]): number {
+        const wrappedCallback = function() {
+            try {
+                if (typeof callback === 'function') {
+                    return callback.apply(this, args);
+                } else if (typeof callback === 'string') {
+                    return eval(callback);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)